src/client/components/Container/index.tsx

```diff
@@ -16,17 +17,20 @@ export default function Container({
   containerId?: string;
 }) {
   return (
-    <div
-      className={classNames(
-        styles.container,
-        { [styles.small]: size === 'small' },
-        { [styles.large]: size === 'large' },
-        className,
-      )}
-      style={style}
-      id={containerId}
-    >
-      {children}
-    </div>
+    <>
+      <div
+        className={classNames(
+          styles.container,
+          { [styles.small]: size === 'small' },
+          { [styles.large]: size === 'large' },
+          className,
+        )}
+        style={style}
+        id={containerId}
+      >
+        {children}
+      </div>
+      <InkeepChatButton />
```

Comment:
This is not a good place, there is no guarantee that `Container` is used on every page. I think the button should be rendered inside `Analytics.tsx`. just like other external services.

Reply:
Makes sense. Moved it!